sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ClientSideRequestStatistics.java

```diff
@@ -329,6 +330,14 @@ public void serialize(
         Instant endTimeUTC;
         @JsonSerialize
         String targetEndpoint;
+        @JsonSerialize
+        String errorMessage;
```

Comment:
Though you don't need `errorMessage` and `inflightRequest`, because the exception should just be happening because of `endTimeUTC` - but it is fine since you have already included it.

Reply:
ACK - just tried to make merging easier for Naveen - and it shouldn't cause any harm here